extensions/imagine/BaseImage.php

```diff
@@ -256,4 +255,25 @@ public static function frame($filename, $margin = 20, $color = '666', $alpha = 1
 
         return $image;
     }
+
+    public static function binary($string)
```

Comment:
@samdark, done for now. Didn't add them yesterday since wanted to know the opinion on the overall idea and approach. ;-)